ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2

```diff
@@ -0,0 +1,61 @@
+{% if snmp_server is defined and snmp_server is not none %}
+### SNMP Configuration Summary
+
+| Contact | Location | Local Interface [VRF] | SNMP Traps | VRF | IPv4 ACL | IPv6 ACL |
+| ------- | -------- | --------------------- | ---------- | --- | -------- | -------- |
+| {{ snmp_server.contact | default('-') }} | {{ snmp_server.location | default('-') }} | {% if snmp_server.local_interfaces is defined and snmp_server.local_interfaces is not none %}{% for interfaces in snmp_server.local_interfaces %}- {{interfaces.name}} [{{interfaces.vrf | default('default')}}]<br>{% endfor %}{% endif %} | {% if snmp_server.traps.enable is defined and snmp_server.traps.enable is not none %}{% if snmp_server.traps.enable %} Enabled {% else %} Disabled {% endif %}{% endif %} | {% if snmp_server.vrfs is defined and snmp_server.vrfs is not none %}{% for vrf in snmp_server.vrfs %}{% if vrf.enable %}- {{vrf.name}}<br>{% endif %}{% endfor %}{% endif %} | {{ snmp_server.access_list_v4 | default('-') }} | {{ snmp_server.access_list_v6 | default('-') }} |
```

Comment:
@aphillipps. Is `<br>` a method you agree on in our documentation engine ?

Reply:
@titom73 @aphillipps I'd say html tags are valid. Even though avoiding them is preferred for readability, I do not see that a big problem here.